### DISCUSSION TITLE
Bug Report: Only part of the hint can be displayed in a Simplefied Chinese context

### DISCUSSION THREAD

**Discussion by aestilog** [community/community#7391] [Copilot Conversations] [answered]
> Windows 10 professional 21H1 19043.1320
> PyCharm 2021.2.3 (Professional Edition)

When I try to get some hints in Chinese, like:
![图片](https://user-images.githubusercontent.com/37444696/140602357-1019f7c7-621e-4d4a-a049-38ef694bb787.png)
Only part of the hint is displayed.

And the I press tab, it becomes:
![图片](https://user-images.githubusercontent.com/37444696/140602385-8bccfe03-b97d-4303-8e06-e5d84a5136a4.png)

Also happens in any other Jetbrains IDEs, but works well in vscode.

**Comment by jansorg** [comment, marked as answer]
Thanks for reporting! Most likely the font width isn't properly calculated. We'll track this internally.
